android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java

```diff
@@ -0,0 +1,49 @@
+package com.reactnative.googlecast.types;
+
+import androidx.annotation.Nullable;
+
+import com.google.android.gms.cast.HlsSegmentFormat;
+
+public class RNGCMediaHlsSegmentFormat {
+  public static int fromJson(final @Nullable String value) {
+    switch (value) {
+      case "aac":
```

Comment:
Since these are abbreviations already, I'd just use the name as the string value, e.g. `AAC`, `AC-3`, `E-AC-3`, `fMP4`, MP3`, `TS`, `TS-AAC` (is this even a thing?), and `MPEG-TS`.

Reply:
What do you mean? That the strings in lowercase should be in uppercase instead? I simply used the string values of the constants.

Reply:
Yeah, I'd make the strings uppercase since they're like proper names, not actual words.

Reply:
These values should map to the string enums defined in the typescript files, right?